### DISCUSSION TITLE
Resource replication doesn't replicate Trix field?

### DISCUSSION THREAD

**Discussion by sutcliffe121** [laravel/nova-issues#5104] [Q&A] [answered]
Hi all. My resource includes various text fields and a trix field. When using the replicate resource feature, the content of the trix field is not replicated. My text fields are populated, but the trix field is empty.

The documentation says 'while using all the data from an existing resource' and 'with all of the replicated resource's data hydrated into the form and ready for tweaking' which suggests I could expect the trix field to be replicated - https://nova.laravel.com/docs/4.0/resources/#resource-replication

Wondering if this is a bug, or intentional that a trix field contents is not replicated?

**Comment by crynobone** [comment, marked as answer]
> Wondering if this is a bug, or intentional that a trix field contents is not replicated?

Replication doesn't automatically replicate fields implementing `Laravel\Nova\Contracts\Deletable` which includes `Trix`. You would need to manually replicate the value and handle "attachments" manually.
